src/data/sym/sym2.lean

```diff
@@ -80,6 +80,22 @@ def sym2 (α : Type u) := quotient (sym2.rel.setoid α)
 
 namespace sym2
 
+attribute [elab_as_eliminator]
+protected lemma ind {f : sym2 α → Prop} (h : ∀ x y, f ⟦(x, y)⟧) : ∀ i, f i :=
+quotient.ind $ prod.rec $ by exact h
```

Comment:
I think if you look for `quotient.ind $ prod.rec` in this file you'll find a few places that can switch to this new definition